core/pom.xml

```diff
@@ -93,6 +93,16 @@
       <version>${mockito.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.typesafe.scala-logging</groupId>
+      <artifactId>scala-logging_${scala.binary.version}</artifactId>
+      <version>3.9.5</version>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
```

Comment:
@nfx need your input here

Reply:
revert to log4j.

Reply:
I didn't see that we were using log4j - at least not in the pom. I can change to use that.